### ISSUE TITLE
Initial support for manual forecasting 

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#355]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Add support for manual forecasting.

### Description
Manual forcasting can be supported by mirroring the functionality implemented as part of #353. We would have the following artifacts and functions

- The confirm week key would change into a "Forecast week" button for weeks in the future. 
- Forecasting a week would add a row to a forcasted weeks table, same way a confirm would add a row to a confirmed week table
- Forecasted hours could be added to a "forecasted timeentries" table
- This would allow us, if we want, to show a "you forcasted X and your actual hours were Y"
- For version 1 we'll add a view "Forecast" to the Reports page

### Status
🧪 Testing

_@okms will create a team channel for a test group. This can be included in milestone 7 if testing goes well, otherwise we need to revert https://github.com/Puzzlepart/did/commit/5542526e979004591c6f1f48ae95f15191136bef._

### Feedback from testing
- [x] Minimum hours for forecast (min 40 hours) @Remi749 @ThomasGranheim 

### Todos
- [x] Have the confirm button be visible but disabled until all hours are confirmed
- [x] Decide what to do with forecasted time entries and periods when confirming the period - _We'll keep them for now_ @okms 

_Originally posted by @okms in https://github.com/Puzzlepart/did365/issues/189#issuecomment-628667476_
